components/list/list-item-mixin.js

```diff
@@ -354,6 +386,19 @@ export const ListItemMixin = superclass => class extends LocalizeCoreElement(Lis
 		});
 	}
 
+	scrollTo() {
```

Comment:
So `scrollTo()` is already a [built-in DOM method](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo) on all elements. Probably don't want to override it?

Reply:
Yeah, maybe `scrollToItem`?  I'll wait until I hear back from @alexli01 re the API in case there are other changes to make.